sycl/plugins/level_zero/pi_level_zero.hpp

```diff
@@ -509,9 +509,11 @@ struct _pi_ze_event_list_t {
                                          const pi_event *EventList,
                                          pi_queue CurQueue);
 
-  // Release all the events in this object's PiEventList, and destroy
-  // the data structures it contains.
-  pi_result releaseAndDestroyPiZeEventList();
+  // Add all the events in this object's PiEventList to the end
+  // of the list EventsToBeReleased. Destroy pi_ze_event_list_t data
+  // structure fields making it look empty.
+  pi_result
+  releaseAndDestroyPiZeEventList(std::list<pi_event> &EventsToBeReleased);
```

Comment:
NIT: maybe reflect the new "collect" semantics in the name?

Reply:
Updated the name to collectEventsForReleaseAndDestroyPiZeEventList.